client/src/app/pages/archetypes/archetypes-page.tsx

```diff
@@ -206,15 +213,47 @@ const Archetypes: React.FC = () => {
           return archetype?.name ?? "";
         },
       },
+      {
+        key: "application.name",
+        title: t("terms.applicationName"),
+        type: FilterType.multiselect,
+        logicOperator: "OR",
+        selectOptions: [
+          ...new Set(
+            archetypes.flatMap(
+              (archetype) =>
+                archetype?.applications
+                  ?.map((app) => app.name)
+                  .filter(Boolean) || []
+            )
+          ),
+        ].map((applicationName) => ({
+          key: applicationName,
+          value: applicationName,
```

Comment:
Having the key and the value for the `selectOptions` be the same is currently annoying me for no good reason.  I mean as long as the key value matches up with what is returned from `getItemValue()` all is well.